code/modules/antag_contest/contest_verbs.dm

```diff
@@ -0,0 +1,225 @@
+/*
+ * Client verbs for the antag contest. Disable this file when the contest is over!
+ * AUG2016
+ */
+
+ #define INDEP		0
+ #define PRO_SYNTH	1
+ #define ANTI_SYNTH	2
+
+// Displays a simple little guide.
+/client/verb/contest_help()
+	set name = "Contest Help"
+	set category = "Contest"
+	set desc = "Information about the contest."
+
+	if (!config.antag_contest_enabled)
+		src << "<span class='warning'>The contest isn't running yet!</span>"
+		return
+
+	var/help = file2text('ingame_manuals/antag_contest.html')
+
+	if (!help)
+		src << "<span class='warning'>Unable to open the document required! Please contact administration or a coder!</span>"
+		return
+
+	src << browse(help, "window=antag_contest_help;size=600x500")
+
+/client/verb/contest_my_characters()
+	set name = "My Character Status"
+	set category = "Contest"
+	set desc = "Displays information to you about your characters, and their standings."
+
+	if (!config.antag_contest_enabled)
+		src << "<span class='warning'>The contest isn't running yet!</span>"
+		return
+
+	if (!establish_db_connection(dbcon))
+		src << "<span class='warning'>Failed to establish SQL connection! Contact a member of staff!</span>"
+		return
+
+	var/DBQuery/character_query = dbcon.NewQuery("SELECT id, name FROM ss13_characters WHERE ckey = :ckey AND deleted_at IS NULL")
+	character_query.Execute(":ckey" = src.ckey)
+	var/list/char_ids = list()
+
+	while (character_query.NextRow())
+		char_ids[character_query.item[1]] = list("name" = character_query.item[2], "assigned" = 0, "side_str" = null, "side_int" = INDEP)
+
+	if (!char_ids.len)
+		src << "<span class='warning'>Something went horribly wrong! Apparently you don't have any saved characters?</span>"
+		return
+
+	var/DBQuery/participation_query = dbcon.NewQuery("SELECT character_id, contest_side FROM ss13_contest_participants WHERE character_id IN :char_ids")
+	participation_query.Execute(":char_ids" = char_ids)
+
+	while (participation_query.NextRow())
+		char_ids[participation_query.item[1]]["side_str"] = participation_query.item[2]
+		char_ids[participation_query.item[1]]["assigned"] = 1
+		// Lazy and convoluted, but I give 0 shits right now.
+		switch (participation_query.item[2])
+			if ("pro_synth")
+				char_ids[participation_query.item[1]]["side_int"] = PRO_SYNTH
+			if ("anti_synth")
+				char_ids[participation_query.item[1]]["side_int"] = ANTI_SYNTH
+
+	var/data = "<center><b>Welcome to the character setup screen!</b></center>"
+	data += "<br><center>Here is the list of your characters, and their allegience</center><hr>"
+
+	for (var/char_id in char_ids)
```

Comment:
Apparently, this time, it goes through values and not indexes. I wish I could dictate which it did. Oh well, easy fix.